package.json

```diff
@@ -51,5 +51,8 @@
     "vue-router": "3.0.1",
     "vue-slick": "1.1.15",
     "vue-template-compiler": "2.6.10"
+  },
+  "dependencies": {
+    "vue-material-design-icons": "^4.11.0"
```

Comment:
Currently, vue-phoenix doesn't depend on vuetify, and avoids making decisions around how things look. Perhaps we can organize the code in a way that a component in vue-phoenix handles the logic, but it provides slots/scoped-slots for the application to use to control the UI. This way we would avoid using vuetify directly in vue-phoenix?